--- node_modules/make-fetch-happen/index.js	1985-10-26 01:15:00.000000000 -0700
+++ index.js	2019-01-20 00:47:07.000000000 -0800
@@ -254,12 +254,7 @@
       }
 
       if (condRes.status === 304) { // 304 Not Modified
-        condRes.body = cachedRes.body
-        return opts.cacheManager.put(req, condRes, opts)
-          .then(newRes => {
-            newRes.headers = new fetch.Headers(revalidatedPolicy.policy.responseHeaders())
-            return newRes
-          })
+        return cachedRes;
       }
 
       return condRes
